tests/lib/fakeinstaller/main.go

```diff
@@ -354,13 +412,27 @@ func run(seedLabel, bootDevice, rootfsCreator string) error {
 }
 
 func main() {
+	if len(os.Args) != 4 {
+		// XXX: allow installing real UC without a classic-rootfs later
+		fmt.Fprintf(os.Stderr, "need seed-label, target-device and classic-rootfs as argument\n")
+		os.Exit(1)
+	}
+	os.Setenv("SNAPD_DEBUG", "1")
```

Comment:
This is temporary, I guess?

Reply:
Removed, it would take it from the environment anyway.